test/suite/wthooks.py

```diff
@@ -367,3 +376,12 @@ def getTimestamp(self):
             except NotImplementedError:
                 pass
         raise Exception('getTimestamp: no implementation')  # should never happen
+
+    def getTierSharePercent(self):
+        """The tier share generator for this test case."""
```

Comment:
I'd call this `tier share value` rather than generator.

Reply:
Fixed